distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -954,7 +958,11 @@ type Column
                 new_name = naming_helper.function_name "ceil" [self]
                 self.rename new_name
             False ->
-                apply_unary_operation self UnaryRoundOperation.CEIL_INSTANCE
+                case self.inferred_precise_value_type.is_decimal of
+                    True ->
+                        apply_unary_operation self UnaryDecimalRoundOperation.CEIL_INSTANCE
+                    False ->
+                        apply_unary_operation self UnaryRoundOperation.CEIL_INSTANCE
```

Comment:
Is there any reason why we cannot merge `UnaryDecimalRoundOperation` with `UnaryRoundOperation`?

I thought the whole idea behind the (relatively new) `UnaryXyzOperation` architecture was to decouple operations from storages.

Reply:
But maybe I misunderstood the `UnaryXyzOperation` approach. If you think that splitting them is better, then I think it is okay to keep this as-is.

Reply:
I think they should be merged, but I was trying to preserve the existing structure until I was clear about the new plan. I have another task #10904 that will require merging these two, so if it's okay I'll merge them in that PR (which will happen as soon as this and #10344 are done).